pkg/storage/storage_test.go

```diff
@@ -526,6 +526,106 @@ var _ = Describe("In-memory storage", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(containerMetrics[0]).NotTo(BeNil())
 	})
+
+	It("should skip metrics when store received older metrics", func() {
+		By("storing a non-empty batch")
+		storage.Store(batch)
+
+		By("construct older data")
+		prevTS := now.Add(-10 * time.Second)
+		olderTs := now.Add(-5 * time.Second)
+		olderBatch := &MetricsBatch{
```

Comment:
dumping whole state of batch makes it hard to spot the difference in code. Please look at other tests to see that they do just minimal change to batch. 

Still I think those tests can be simplified. I will take a look into this.